techdocs/docs/using-the-package/apis-&-components/sanitize.md

```diff
@@ -47,7 +47,7 @@ console.log(sanitized4); // Output: { "": "this.field == 'value'" }
 const sanitized5 = sanitize(input1, {
   removeHTMLTags: true,
   removeSQLInjectionPatterns: true,
-  removeJavaScriptCode: true,
+  removeScriptTags: true,
   removeNoSQLInjectionPatterns: true,
 });
 console.log(sanitized5); // Output: Safe content
```

Comment:
this line is fine, but noting here that the note below is not formatted correctly